tests/compilation/test_predictor_rl.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import annotations
 
 import os
+from pathlib import Path
 
 import pytest
 from qiskit import QuantumCircuit
 
-from mqt.bench import get_benchmark
+from mqt.bench import benchmark_generator, get_benchmark
```

Comment:
Is it really necessary to import `benchmark_generator`? I do not see the reason for just using `get_benchmark`.